layouts/default.vue

```diff
@@ -75,6 +75,7 @@ html {
   display: inherit;
   left: 0;
   transition: 0.3s ease-out;
+  @apply left-0 bg-mtl-white;
```

Comment:
@f3ltron I think we need to add an empty line above like you did in `.card-block`

```JS
.card-block {
  box-shadow: 0 8px 42px rgba(0, 0, 0, 0.08);

  @apply px-4 pb-2 border border-mtl-black-200 rounded-lg relative;
}
```

Reply:
if not just resolve and merge 👍

Reply:
we dont have the css lint on master yet. We will follow it after